<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="46028" started-at="2020-01-29T03:43:38Z" finished-at="2020-01-29T03:44:24Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="46028" started-at="2020-01-29T03:43:38Z" finished-at="2020-01-29T03:44:24Z">
      <class name="driverFactory.NewTest">
        <test-method status="PASS" signature="adminLogin()[pri:0, instance:driverFactory.NewTest@26653222]" name="adminLogin" is-config="true" duration-ms="15043" started-at="2020-01-29T09:13:38Z" finished-at="2020-01-29T09:13:53Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- adminLogin -->
        <test-method status="FAIL" signature="f()[pri:0, instance:driverFactory.NewTest@26653222]" name="f" duration-ms="19605" started-at="2020-01-29T09:13:53Z" finished-at="2020-01-29T09:14:13Z">
          <exception class="java.io.FileNotFoundException">
            <message>
              <![CDATA[.\TestOutput\Hybrid.xlsx (The system cannot find the path specified)]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.io.FileNotFoundException: .\TestOutput\Hybrid.xlsx (The system cannot find the path specified)
	at java.io.FileOutputStream.open0(Native Method)
	at java.io.FileOutputStream.open(FileOutputStream.java:270)
	at java.io.FileOutputStream.<init>(FileOutputStream.java:213)
	at java.io.FileOutputStream.<init>(FileOutputStream.java:101)
	at Utilities.ExcelFileUtils.setData(ExcelFileUtils.java:118)
	at driverFactory.NewTest.f(NewTest.java:42)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
            </full-stacktrace>
          </exception> <!-- java.io.FileNotFoundException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- f -->
        <test-method status="PASS" signature="logut()[pri:0, instance:driverFactory.NewTest@26653222]" name="logut" is-config="true" duration-ms="11312" started-at="2020-01-29T09:14:13Z" finished-at="2020-01-29T09:14:24Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- logut -->
      </class> <!-- driverFactory.NewTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
